scanner/src/main/kotlin/scanners/fossid/FossIdUrlProvider.kt

```diff
@@ -66,7 +66,15 @@ internal class FossIdUrlProvider private constructor(
          */
         fun create(
             urlMapping: Collection<String> = emptyList()
-        ): FossIdUrlProvider = FossIdUrlProvider(urlMapping.toRegexMapping())
+        ): FossIdUrlProvider {
+            val mappings = urlMapping.toRegexMapping()
+            logger.info { "${mappings.size} mappings have been loaded." }
```

Comment:
Do we really need this if we have the following? I.e. do we expect there to be so many mappings that it's too hard to count them manually (eventually by looking at log line numbers)?